app/controllers/application_controller.rb

```diff
@@ -1,21 +1,16 @@
 require 'pry'
 require './config/environment'
+require './app/services/weather_service'
 
 class ApplicationController < Sinatra::Base
+  get '/api/v1/weather' do
+    weather_days = WeatherFacade.daily_weather(params)
 
-  get "/api/v1/weather" do
-    # params = {"lat"=>"35", "lon"=>"139"}
-    lat = params[:lat]
-    lon = params[:lon]
-    conn = Faraday.new(url: "http://api.openweathermap.org")
-    
-    response = conn.get("/data/2.5/onecall") do |req|
-      req.params["lat"] = lat
-      req.params["lon"] = lon
-      req.params["appid"] = ENV['WEATHER_API_KEY']
+    if weather_days.instance_of?(Array)
+      JSON WeatherSerializer.new(weather_days)
+    else
+      JSON weather_days
+      # "{\"cod\":\"400\",\"message\":\"wrong longitude\"}"
```

Comment:
I'm wondering if there's a reason this is commented out?

Reply:
originally, this was here so people knew the structure of what was to be expected in the API response. But now that we've updated the Readme with that info.  I think it's ok to remove the comments. We've now done so.  thanks!